cpu/cc26x0/vendor/Makefile

```diff
@@ -1,6 +1,5 @@
 MODULE=cc26x0_vendor
 
 INCLUDES +=-I$(CURDIR)/inc
-DIRS = driverlib inc
 
```

Comment:
You want to move that whole Makefile to `vendor/driverlib`, then add `vendor/driverlib` to `dirs` in `cc26x0/Makefile`